EIPS/eip-1.md

```diff
@@ -183,7 +183,9 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 #### `requires` header
 
-EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
+EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on. If such a dependency exists, this field is required.
+
+A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency.
```

Comment:
```suggestion
A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Mentioning another EIP within the Specification section automatically creates a dependency. Mentioning another EIP outside of the Specification section does not necessarily create a dependency.
```
My personal preference is this.

Reply:
I think the existing text is fine. We don't need to spoon feed authors.

Reply:
We need to spoon-feed bots, though. @SamWilsn can decide whether this wording change is valuable or not.

Reply:
We can keep the general text that @lightclient prefers, and create a rule in `eipw` to enforce @Pandapip1's rule.